### PR TITLE
Create YYRC-EAGLEF722

### PR DESCRIPTION
**Pull request by tianv2mu0897**
Hello, I would like to create a branch and look forward to merging it; As I am creating it for the first time, if there are any issues, I am very much looking forward to you telling me the problem points; Thank you!

### CODE REVIEW

File: configs/YYRC-EAGLEF722V0 [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        EAGLEF722
+#define MANUFACTURER_ID   YYRC
```
Comment (nerdCopter) [inline configs/YYRC-EAGLEF722V0:27]: `YYRC` will need to be added to https://github.com/betaflight/config/blob/master/Manufacturers.md

**Comment by nerdCopter** [comment]
@tianv2mu0897 , may need to provide schematics and samples to dev-team.  may need to join Discord and contact @sugaarK for such.

File: configs/YYRC-EAGLEF722V0 (hunk 1 above)
Comment (nerdCopter) [reply configs/YYRC-EAGLEF722V0:27]: https://github.com/betaflight/config/pull/463

File: configs/YYRC-EAGLEF722V0 [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 115
+#define BEEPER_INVERTED
+#define MAX7456_SPI_INSTANCE SPI3
+#define PINIO1_CONFIG 129
+#define PINIO2_CONFIG 129
+#define PINIO1_BOX 0
+#define PINIO2_BOX 40
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722V0:108]: No PINIO pins defined.

```suggestion
```

File: configs/YYRC-EAGLEF722V0 [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    TIMER_PIN_MAP( 1, PC8 , 2,  0) \
+    TIMER_PIN_MAP( 2, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 3, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 4, PA9 , 1,  0) \
+    TIMER_PIN_MAP( 6, PB3 , 1,  0) \
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define USE_BARO
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722V0:94]: Duplicate define
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by tianv2mu0897** [comment]
hello:
&nbsp; &nbsp; &nbsp; &nbsp; I have updated it;


【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月18日(星期二) 晚上11:31
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis requested changes on this pull request.
 
 
In configs/YYRC-EAGLEF722V0:
 &gt; +#define PINIO1_CONFIG 129 +#define PINIO2_CONFIG 129 +#define PINIO1_BOX 0 +#define PINIO2_BOX 40  
No PINIO pins defined.
 ⬇️ Suggested change -#define PINIO1_CONFIG 129 -#define PINIO2_CONFIG 129 -#define PINIO1_BOX 0 -#define PINIO2_BOX 40  
 
In configs/YYRC-EAGLEF722V0:
 &gt; +#define GYRO_1_CS_PIN PA0 +#define USB_DETECT_PIN PA10 +#define PULLDOWN_PIN_UART1 PC0 + +#define TIMER_PIN_MAPPING \ +    TIMER_PIN_MAP( 0, PB9 , 2, -1) \ +    TIMER_PIN_MAP( 1, PC8 , 2,  0) \ +    TIMER_PIN_MAP( 2, PC9 , 2,  0) \ +    TIMER_PIN_MAP( 3, PA8 , 1,  0) \ +    TIMER_PIN_MAP( 4, PA9 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0) \ + +#define ADC1_DMA_OPT        0 + +#define MAG_I2C_INSTANCE (I2CDEV_1) +#define USE_BARO  
Duplicate define
 ⬇️ Suggested change -#define USE_BARO

File: configs/YYRC-EAGLEF722V0 (hunk 3 above)
Comment (haslinghuis) [reply configs/YYRC-EAGLEF722V0:94]: @tianv2mu0897 please correct.

**Comment by tianv2mu0897** [comment]
YES,Is OK.





------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Wed,Jun 19,2024 1:05 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis commented on this pull request.
 
 
In configs/YYRC-EAGLEF722V0:
 &gt; +#define GYRO_1_CS_PIN PA0 +#define USB_DETECT_PIN PA10 +#define PULLDOWN_PIN_UART1 PC0 + +#define TIMER_PIN_MAPPING \ +    TIMER_PIN_MAP( 0, PB9 , 2, -1) \ +    TIMER_PIN_MAP( 1, PC8 , 2,  0) \ +    TIMER_PIN_MAP( 2, PC9 , 2,  0) \ +    TIMER_PIN_MAP( 3, PA8 , 1,  0) \ +    TIMER_PIN_MAP( 4, PA9 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0) \ + +#define ADC1_DMA_OPT        0 + +#define MAG_I2C_INSTANCE (I2CDEV_1) +#define USE_BARO  
@tianv2mu0897 please correct.

File: configs/YYRC-EAGLEF722V0 [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+#define SPI2_SDO_PIN PB15
+#define SPI3_SDO_PIN PB5
+#define ADC_VBAT_PIN PB0
+#define ADC_CURR_PIN PC3
+#define FLASH_CS_PIN PB12
+#define MAX7456_SPI_CS_PIN PA15
+#define GYRO_1_EXTI_PIN PC4
+#define GYRO_1_CS_PIN PA0
+#define USB_DETECT_PIN PA10
+#define PULLDOWN_PIN_UART1 PC0
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722V0:81]: Sorry found one more ;) This pin does not exist.
```suggestion
```

**Comment by tianv2mu0897** [comment]
OK





------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Wed,Jun 19,2024 6:43 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)

**Comment by tianv2mu0897** [comment]
It is update





------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Wed,Jun 19,2024 6:43 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis commented on this pull request.
 
 
In configs/YYRC-EAGLEF722V0:
 &gt; +#define SPI2_SCK_PIN PB13 +#define SPI3_SCK_PIN PC10 +#define SPI1_SDI_PIN PA6 +#define SPI2_SDI_PIN PB14 +#define SPI3_SDI_PIN B4 +#define SPI1_SDO_PIN PA7 +#define SPI2_SDO_PIN PB15 +#define SPI3_SDO_PIN PB5 +#define ADC_VBAT_PIN PB0 +#define ADC_CURR_PIN PC3 +#define FLASH_CS_PIN PB12 +#define MAX7456_SPI_CS_PIN PA15 +#define GYRO_1_EXTI_PIN PC4 +#define GYRO_1_CS_PIN PA0 +#define USB_DETECT_PIN PA10 +#define PULLDOWN_PIN_UART1 PC0  
Sorry found one more ;) This pin does not exist.
 ⬇️ Suggested change -#define PULLDOWN_PIN_UART1 PC0

**Comment by tianv2mu0897** [comment]
OK,


正在使用发音


I will contact @sugaark;thank you.





------------------ Original ------------------
From: nerdCopter ***@***.***&gt;
Date: Tue,Jun 18,2024 10:29 PM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@tianv2mu0897 , may need to provide schematics and samples to dev-team.  may need to join Discord and contact @sugaarK for such.

File: configs/YYRC-EAGLEF722 [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    TIMER_PIN_MAP( 1, PC8 , 2,  0) \
+    TIMER_PIN_MAP( 2, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 3, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 4, PA9 , 1,  0) \
+    TIMER_PIN_MAP( 6, PB3 , 1,  0) \
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define USE_BARO
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722:94]:
```suggestion
#define BARO_I2C_INSTANCE (I2CDEV_1)
```

**Review by haslinghuis — APPROVED** [review]

File: configs/YYRC-EAGLEF722 [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+#define USB_DETECT_PIN PA10
+#define PULLDOWN_PIN PC0
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PB9 , 2, -1) \
+    TIMER_PIN_MAP( 1, PC8 , 2,  0) \
+    TIMER_PIN_MAP( 2, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 3, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 4, PA9 , 1,  0) \
+    TIMER_PIN_MAP( 6, PB3 , 1,  0) \
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722:89]:
```suggestion
    TIMER_PIN_MAP( 6, PB3 , 1,  0)
```

File: configs/YYRC-EAGLEF722 [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+#define BEEPER_INVERTED
+#define MAX7456_SPI_INSTANCE SPI3
+#define FLASH_SPI_INSTANCE SPI2
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW90_DEG
+#define GYRO_1_ALIGN_YAW 900
+
++#define PINIO2_CONFIG 129
++#define PINIO1_BOX 0
++#define PINIO2_BOX 40
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722:111]:
```suggestion
```

**Comment by tianv2mu0897** [comment]
OK,I will update.Thank you.







------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Thu,Jun 20,2024 5:09 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis commented on this pull request.
 
 
In configs/YYRC-EAGLEF722:
 &gt; +#define GYRO_1_CS_PIN PA0 +#define USB_DETECT_PIN PA10 +#define PULLDOWN_PIN PC0 + +#define TIMER_PIN_MAPPING \ +    TIMER_PIN_MAP( 0, PB9 , 2, -1) \ +    TIMER_PIN_MAP( 1, PC8 , 2,  0) \ +    TIMER_PIN_MAP( 2, PC9 , 2,  0) \ +    TIMER_PIN_MAP( 3, PA8 , 1,  0) \ +    TIMER_PIN_MAP( 4, PA9 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0) \ + +#define ADC1_DMA_OPT        0 + +#define MAG_I2C_INSTANCE (I2CDEV_1) +#define USE_BARO  ⬇️ Suggested change -#define USE_BARO +#define BARO_I2C_INSTANCE (I2CDEV_1)

**Comment by haslinghuis** [comment]
Have made the updates already

**Comment by tianv2mu0897** [comment]
OK,I will update it;





------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Thu,Jun 20,2024 5:11 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis commented on this pull request.
 
 
In configs/YYRC-EAGLEF722:
 &gt; +#define ADC_VBAT_PIN PB0 +#define ADC_CURR_PIN PC3 +#define FLASH_CS_PIN PB12 +#define MAX7456_SPI_CS_PIN PA15 +#define GYRO_1_EXTI_PIN PC4 +#define GYRO_1_CS_PIN PA0 +#define USB_DETECT_PIN PA10 +#define PULLDOWN_PIN PC0 + +#define TIMER_PIN_MAPPING \ +    TIMER_PIN_MAP( 0, PB9 , 2, -1) \ +    TIMER_PIN_MAP( 1, PC8 , 2,  0) \ +    TIMER_PIN_MAP( 2, PC9 , 2,  0) \ +    TIMER_PIN_MAP( 3, PA8 , 1,  0) \ +    TIMER_PIN_MAP( 4, PA9 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0) \  ⬇️ Suggested change -    TIMER_PIN_MAP( 6, PB3 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0)

**Comment by tianv2mu0897** [comment]
OK,I will update it；





------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Thu,Jun 20,2024 5:12 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis commented on this pull request.
 
 
In configs/YYRC-EAGLEF722:
 &gt; + ++#define PINIO2_CONFIG 129 ++#define PINIO1_BOX 0 ++#define PINIO2_BOX 40  ⬇️ Suggested change - -+#define PINIO2_CONFIG 129 -+#define PINIO1_BOX 0 -+#define PINIO2_BOX 40

File: configs/YYRC-EAGLEF722 [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+#define SPI2_SDO_PIN PB15
+#define SPI3_SDO_PIN PB5
+#define ADC_VBAT_PIN PB0
+#define ADC_CURR_PIN PC3
+#define FLASH_CS_PIN PB12
+#define MAX7456_SPI_CS_PIN PA15
+#define GYRO_1_EXTI_PIN PC4
+#define GYRO_1_CS_PIN PA0
+#define USB_DETECT_PIN PA10
+#define PULLDOWN_PIN PC0
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722:81]:
```suggestion
```

**Comment by tianv2mu0897** [comment]
It is be use





------------------ Original ------------------
From: Mark Haslinghuis ***@***.***&gt;
Date: Thu,Jun 20,2024 5:19 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@haslinghuis commented on this pull request.
 
 
In configs/YYRC-EAGLEF722:
 &gt; +#define SPI2_SCK_PIN PB13 +#define SPI3_SCK_PIN PC10 +#define SPI1_SDI_PIN PA6 +#define SPI2_SDI_PIN PB14 +#define SPI3_SDI_PIN B4 +#define SPI1_SDO_PIN PA7 +#define SPI2_SDO_PIN PB15 +#define SPI3_SDO_PIN PB5 +#define ADC_VBAT_PIN PB0 +#define ADC_CURR_PIN PC3 +#define FLASH_CS_PIN PB12 +#define MAX7456_SPI_CS_PIN PA15 +#define GYRO_1_EXTI_PIN PC4 +#define GYRO_1_CS_PIN PA0 +#define USB_DETECT_PIN PA10 +#define PULLDOWN_PIN PC0  ⬇️ Suggested change -#define PULLDOWN_PIN PC0

File: configs/YYRC-EAGLEF722 [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    TIMER_PIN_MAP( 6, PB3 , 1,  0)
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+
+#define ADC_INSTANCE ADC1
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON
```
Comment (ot0tot) [inline configs/YYRC-EAGLEF722:97]: No need to set DShot burst to On. Delete this or set to AUTO. 
Consider adding `#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON`.

**Review by ot0tot — COMMENTED** [review]
Two minor changes suggested, otherwise it looks good!

**Comment by tianv2mu0897** [comment]
OK,I will update it,Thank you!





------------------ Original ------------------
From: ot0tot ***@***.***&gt;
Date: Thu,Jun 20,2024 6:17 AM
To: betaflight/config ***@***.***&gt;
Cc: tianmu0897 ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@ot0tot commented on this pull request.
 
Two minor changes suggested, otherwise it looks good!

 
In configs/YYRC-EAGLEF722:
 &gt; +#define USE_GYRO +#define USE_GYRO_SPI_MPU6000 +#define USE_GYRO_SPI_ICM42688P +#define USE_BARO +#define USE_BARO_BMP280 +#define USE_BARO_DPS310 +#define USE_FLASH +#define USE_FLASH_M25P16 +#define USE_MAX7456 + +#define BEEPER_PIN PC13 +#define MOTOR1_PIN PC9 +#define MOTOR2_PIN PC8 +#define MOTOR3_PIN PA8 +#define MOTOR4_PIN PA9 +#define RX_PPM_PIN PB9  
Does this FC actually have a PPM input pin?
 
 
In configs/YYRC-EAGLEF722:
 &gt; +#define TIMER_PIN_MAPPING \ +    TIMER_PIN_MAP( 0, PB9 , 2, -1) \ +    TIMER_PIN_MAP( 1, PC8 , 2,  0) \ +    TIMER_PIN_MAP( 2, PC9 , 2,  0) \ +    TIMER_PIN_MAP( 3, PA8 , 1,  0) \ +    TIMER_PIN_MAP( 4, PA9 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0) + +#define ADC1_DMA_OPT        0 + +#define MAG_I2C_INSTANCE (I2CDEV_1) +#define BARO_I2C_INSTANCE (I2CDEV_1) + +#define ADC_INSTANCE ADC1 +#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH +#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON  
No need to set DShot burst to On. Delete this or set to AUTO.
 Consider adding #define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON.

**Comment by tianv2mu0897** [comment]
hello:
hello:&nbsp; &nbsp; &nbsp;
&nbsp; &nbsp; &nbsp;i am join in discord,how i will contact he? what is his discord name?can you tell me and his discord name;Thank you!




【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月18日(星期二) 晚上10:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@tianv2mu0897 , may need to provide schematics and samples to dev-team.  may need to join Discord and contact @sugaarK for such.

**Comment by tianv2mu0897** [comment]
hello:
&nbsp; &nbsp; &nbsp; betafligth/config is When can it be merged into BETAFLIGHT！thank you!




【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月20日(星期四) 上午6:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
@ot0tot commented on this pull request.
 
Two minor changes suggested, otherwise it looks good!

 
In configs/YYRC-EAGLEF722:
 &gt; +#define USE_GYRO +#define USE_GYRO_SPI_MPU6000 +#define USE_GYRO_SPI_ICM42688P +#define USE_BARO +#define USE_BARO_BMP280 +#define USE_BARO_DPS310 +#define USE_FLASH +#define USE_FLASH_M25P16 +#define USE_MAX7456 + +#define BEEPER_PIN PC13 +#define MOTOR1_PIN PC9 +#define MOTOR2_PIN PC8 +#define MOTOR3_PIN PA8 +#define MOTOR4_PIN PA9 +#define RX_PPM_PIN PB9  
Does this FC actually have a PPM input pin?
 
 
In configs/YYRC-EAGLEF722:
 &gt; +#define TIMER_PIN_MAPPING \ +    TIMER_PIN_MAP( 0, PB9 , 2, -1) \ +    TIMER_PIN_MAP( 1, PC8 , 2,  0) \ +    TIMER_PIN_MAP( 2, PC9 , 2,  0) \ +    TIMER_PIN_MAP( 3, PA8 , 1,  0) \ +    TIMER_PIN_MAP( 4, PA9 , 1,  0) \ +    TIMER_PIN_MAP( 6, PB3 , 1,  0) + +#define ADC1_DMA_OPT        0 + +#define MAG_I2C_INSTANCE (I2CDEV_1) +#define BARO_I2C_INSTANCE (I2CDEV_1) + +#define ADC_INSTANCE ADC1 +#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH +#define DEFAULT_DSHOT_BURST DSHOT_DMAR_ON  
No need to set DShot burst to On. Delete this or set to AUTO.
 Consider adding #define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON.

**Comment by ot0tot** [comment]
Awaiting schematics and dev samples. Have you joined the Discord and contacted @sugaarK ?

**Comment by tianv2mu0897** [comment]
I have joined the discord,But I didn't get in touch ***@***.***&nbsp;?





【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月28日(星期五) 上午9:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
Awaiting schematics and dev samples. Have you joined the Discord and contacted @sugaarK ?

**Review by sugaarK — APPROVED** [review]

**Comment by tianv2mu0897** [comment]
邮件已收到，谢谢！

File: configs/YYRC-EAGLEF722 [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    TIMER_PIN_MAP( 3, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 4, PA9 , 1,  0) \
+    TIMER_PIN_MAP( 6, PB3 , 1,  0)
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+
+#define ADC_INSTANCE ADC1
```
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722:95]: ADC1 is default
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Need to change filename to match board_name

**Comment by nerdCopter** [comment]
> Need to change filename to match board_name

@tianv2mu0897 , in your repo, (assuming linux)
```
mkdir configs/EAGLEF722
git mv configs/YYRC-EAGLEF722 configs/EAGLEF722/config.h
```
then commit/push

**Comment by nerdCopter** [comment]
https://github.com/betaflight/config/pull/465/commits/83488d46646a1a3fdfcf9f7d6532368d49554f88 is still not proper, it should be `EAGLEF722/config.h`

**Comment by tianv2mu0897** [comment]
hello:
&nbsp; &nbsp; &nbsp;&nbsp;How do I operate on the page?Thank you!




How do I operate on the page

【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月12日(星期五) 晚上9:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
83488d4 is still not proper, it should be EAGLEF722/config.h

**Comment by tianv2mu0897** [comment]
hello:
&nbsp; &nbsp; &nbsp;&nbsp;How do I operate on the page?Thank you!



【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月12日(星期五) 晚上9:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





 
83488d4 is still not proper, it should be EAGLEF722/config.h

**Comment by nerdCopter** [comment]
- not possible via web-interface.
- i fixed and pushed

**Comment by nerdCopter** [comment]
@haslinghuis , please repeat last suggestion from new file. TY

**Comment by tianv2mu0897** [comment]
Thank you very much;




【天v2幕】
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "betaflight/config"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月12日(星期五) 晚上10:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [betaflight/config] Create YYRC-EAGLEF722 (PR #465)





  
not possible via web-interface.
 
i fixed and pushed

File: configs/EAGLEF722/config.h [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+    TIMER_PIN_MAP( 3, PA8 , 1,  0) \
+    TIMER_PIN_MAP( 4, PA9 , 1,  0) \
+    TIMER_PIN_MAP( 6, PB3 , 1,  0)
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+
+#define ADC_INSTANCE ADC1
```
Comment (haslinghuis) [inline configs/EAGLEF722/config.h:95]: ADC1 is default instance
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

File: configs/YYRC-EAGLEF722 (hunk 9 above)
Comment (haslinghuis) [inline configs/YYRC-EAGLEF722]:
```suggestion
#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON
```

**Pull request merged**